src/py21cmfast/src/Constants.h

```diff
@@ -8,220 +8,107 @@
+    const double k_B;       // Boltzmann's constant in erg/K
+    const double m_p;       // proton mass in g
+    const double m_e;       // electron mass in g
+    const double G;         // gravitational constant in cgs
+    const double e_charge;  // electron charge in esu (g^1/2 cm^3/2 s^-1)
+    const double vac_perm;  // permittivity of free space in F/m (C^2 s^2 / kg m^3)
+
+    // Units
+    const double Msun;        // solar mass in g
+    const double s_per_yr;    // seconds per year
+    const double cm_per_Mpc;  // cm per Mpc
+    const double eV_to_Hz;    // convert eV to Hz
+
+    // Photon frequencies and temperatures
+    const double nu_ion_HI;        // ionization frequency of HI in Hz
+    const double nu_ion_HeI;       // ionization frequency of HeI in Hz
+    const double nu_ion_HeII;      // ionization frequency of HeII in Hz
+    const double nu_LW_thresh;     // Lyman-Werner threshold frequency in Hz
+    const double nu_Ly_alpha;      // frequency of Lyman-alpha in Hz
+    const double T_cmb;            // CMB temperature at z=0 in K
+    const double T_21;             // Temperature corresponding to 21cm photon in K
+    const double lambda_21;        // Wavelength of 21cm Radiation in cm
+    const double lambda_Ly_alpha;  // Wavelength of Lyman-Alpha in Angstroms
+    const double lambda_Ly_beta;   // Wavelength of Lyman-Beta in Angstroms
+    const double lambda_Ly_gamma;  // Wavelength of Lyman-Gamma in Angstroms
+
+    // Cross-sections and rate coefficients
+    const double sigma_T;      // Thomson scattering cross section in cm^2
+    const double sigma_HI;     // HI ionization cross section at 13.6 eV in cm^2
+    const double A10;          // spontaneous emission coefficient of 21cm in s^-1
+    const double A_Ly_alpha;   // Spontaneous emission coefficient for Lyman-Alpha line in s^-1
+    const double f_alpha;      // oscillator strength of Lya
+    const double alpha_A_10k;  // case A hydrogen recombination coefficient at 10,000 K in cm^3 s^-1
+    const double alpha_B_10k;  // case B hydrogen recombination coefficient at 10,000 K in cm^3 s^-1
+    const double alpha_B_20k;  // case B hydrogen recombination coefficient at 20,000 K in cm^3 s^-1
+
+    // misc.
+    const double l_factor;       // factor relating cube length to filter radius = (4PI/3)^(-1/3)
+    const double delta_c_sph;    // critical overdensity in spherical collapse model
+    const double delta_c_delos;  // critical overdensity in Delos 2025 random walk model.
+} PhysicalConstants;
+
+extern const PhysicalConstants physconst;
+
+// Below are a few leftover macros, where they were used across multiple files or
+// depended on input parameters.
+
+// BEWARE: Since these macros are defined in a header, they *can* be applied to
+// any code included by 21cmFAST, e.g. if file A includes Constants.h, then includes
+// file B which includes fftw.h, the macros do find/replaces on fftw.h during compilation.
```

Comment:
Thanks for adding the explanation. I wonder if we shouldn't 'namespace' our own macros... like give each a prefix `_21CMFAST_`?